composites/Plugin/CornerstoneContent/components/CornerstoneToggle.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Styled from "styled-components";
+import noop from "lodash/noop";
+
+import GutenbergToggle from "../../Shared/components/GutenbergToggle";
+
+const Cornerstone = Styled.div`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    
+    label { 
+        margin-right: 10px;
+        flex-shrink: 0;
+        max-width: 75%;
+    }
+`;
+
+class CornerstoneToggle extends React.Component {
+
```

Comment:
eslint "warning  Block must not be padded by blank lines", please remove this blank line